plugins/modules/ecs_service.py

```diff
@@ -188,6 +188,14 @@
         required: false
         choices: ["DAEMON", "REPLICA"]
         type: str
+    enable_execute_command:
+        description:
+          - Whether or not to enable the execute command functionality for the containers in this task.
+          - If `true`, this enables execute command functionality on all containers in the task.
+          - This option requires botocore >= 1.20.28
```

Comment:
```suggestion
          - Whether or not to enable the execute command functionality for the containers in the ECS task.
          - If I(enable_execute_command=true) execute command functionality is enabled on all containers in the ECS task.
          - This option requires botocore >= 1.20.28.
```
Slight formatting tweaks.

Because 'task' can have multiple meanings in this context I'd suggest being more specific:
- task == Ansible Playbook entry (ie how most people will run this module)
- task == ECS task